x/ccv/consumer/keeper/relay.go

```diff
@@ -110,6 +123,16 @@ func (k Keeper) SendVSCMaturedPackets(ctx sdk.Context) error {
 				return err
 			}
 			k.DeletePacketMaturityTime(ctx, vscId)
+			ctx.EventManager().EmitEvent(
+				sdk.NewEvent(
+					ccv.EventTypeSendMaturedVSCPacket,
+					sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
+					sdk.NewAttribute(ccv.AttributeChainID, ctx.ChainID()),
```

Comment:
I don't think we need the chain ID given that the user already needs to subscribe to the chain to get this event.

Reply:
Having a chainID visible could be useful.

The user could be listening to events from multiple consumer chains in parallel and merge them into a single processing pipeline

In that case it is alot easier to manage if you already have a chain ID